### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by diwakergupta**
Mostly standard bug and feature issue templates with some minor tweaks.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
```
Comment (jcnelson) [inline .github/ISSUE_TEMPLATE/bug_report.md:14]: There won't be much clicking or scrolling here.  I recommend we ask for the list of commands executed to reproduce.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
```
Comment (jcnelson) [inline .github/ISSUE_TEMPLATE/bug_report.md:23]: I'd ask for stack traces, error messages, and log output indicating the error (as well as debug log output).  Screenshots aren't really appropriate here.

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
```
Comment (jcnelson) [inline .github/ISSUE_TEMPLATE/feature_request.md:6]: I'd recommend dropping `assignees` and `labels`.  That's more our job.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (kantai) [reply .github/ISSUE_TEMPLATE/bug_report.md:23]: I would also add -- "If you are debugging a smart contract, please include the smart contract code that demonstates the issue"

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 3 above)
Comment (diwakergupta) [reply .github/ISSUE_TEMPLATE/feature_request.md:6]: These fields are not visible when creating an issue, only relevant for API calls. Omitting them here is the same thing as setting to empty values.

**Pull request merged**